security.rst

```diff
@@ -999,13 +999,14 @@ look like::
 
     use Symfony\Component\Security\Core\User\UserInterface;
 
-    public function indexAction(UserInterface $user)
+    public function indexAction()
     {
         if (!$this->get('security.authorization_checker')->isGranted('IS_AUTHENTICATED_FULLY')) {
             throw $this->createAccessDeniedException();
         }
```

Comment:
Bah, now that I look a bit more, this is a bit of a mess :/ still. I'd like to propose a few more changes:

1) In the above code, use the simpler `$this->denyAccessUnlessGranted()`.
2) Below, user the simpler `$user = $this->getUser()`.

3) Move the suggestion about UserInterface to below, and I'll suggest a wording tweak too :)
```php
$user = $this->getUser();
// or you can also type-hint a method argument with UserInterface: e.g. "UserInterface $user"
```

4) Below (you need to expand the code-blocks to see it), there is a `versionadded:: 3.2`. I think we should shorten that quite a bit:

```rst
 .. versionadded:: 3.2
     The ability to get the user by type-hinting an argument with UserInterface
     was introduced in Symfony 3.2.
```